catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/MtirpbAttribute.java

```diff
@@ -34,112 +35,124 @@
 
     private static final List<NitfAttribute<Tre>> ATTRIBUTES = new LinkedList<>();
 
-    private static final String ATTRIBUTE_NAME_PREFIX = "mtirpb.";
+    private static final String PREFIX = ExtNitfUtility.EXT_NITF_PREFIX + "mtirpb.";
+
+    public static final String NUMBER_OF_VALID_TARGETS = PREFIX + "number-of-valid-targets";
+
+    public static final String AIRCRAFT_LOCATION = PREFIX + "aircraft-location";
+
+    public static final String AIRCRAFT_ALTITUDE = PREFIX + "aircraft-altitude";
+
+    public static final String AIRCRAFT_ALTITUDE_UNITS = PREFIX + "aircraft-altitude-unit-of-measure";
+
+    public static final String AIRCRAFT_HEADING = PREFIX + "aircraft-heading";
+
+    public static final String COSINE_OF_GRAZE_ANGLE = PREFIX + "cosine-of-graze-angle";
+
+    public static final String DESTINATION_POINT = PREFIX + "destination-point";
+
+    public static final String MTI_LR = PREFIX + "mti-left-or-right";
+
+    public static final String PATCH_NUMBER = PREFIX + "patch-number";
+
+    public static final String SCAN_DATE_AND_TIME = PREFIX + "scan-date-and-time";
+
+    public static final String SQUINT_ANGLE = PREFIX + "squint-angle";
+
+    public static final String WIDE_AREA_MTI_FRAME_NUMBER = PREFIX + "wide-area-mti-frame-number";
+
+    public static final String WIDE_AREA_MTI_BAR_NUMBER = PREFIX + "wide-area-mti-bar-number";
 
     /*
      * Normalized attributes. These taxonomy terms will be duplicated by `ext.nitf.mtirpb.*` when
      * appropriate
      */
 
-    public static final MtirpbAttribute NUMBER_OF_VALID_TARGETS =
+    public static final MtirpbAttribute NUMBER_OF_VALID_TARGETS_ATTRIBUTE =
             new MtirpbAttribute(Isr.TARGET_REPORT_COUNT,
-                    "NO_VALID_TARGETS",
+                    NUMBER_OF_VALID_TARGETS,
```

Comment:
It seems that everywhere else the second param here matches the second param to the `TreUtility.getTreValue()` function. This one differs.